docs/guides/Catalog/managing-skus.mdx

```diff
@@ -0,0 +1,517 @@
+---
+title: "Managing SKUs"
+slug: "managing-skus"
+excerpt: "Learn how to create, update, and activate an SKU with the Catalog API."
+hidden: false
+createdAt: "2021-12-14T13:56:04.502Z"
+updatedAt: "2022-02-04T21:26:33.803Z"
+---
+
+This step-by-step guide will walk you through the process of creating, updating, and activating a [Stock Keeping Unit (SKU)](https://help.vtex.com/en/tracks/catalog-101--5AF0XfnjfWeopIFBgs3LIQ/3mJbIqMlz6oKDmyZ2bKJoA#) within the VTEX platform. An SKU represents a specific item in the inventory that customers can purchase, essentially serving as a unique product variation, such as a "long sleeve gray size S shirt."
+
+<CH.Spotlight>
+
+## Before you begin
```

Comment:
Aqui achei estranho o cURL aparecer ao lado do Before you begin:

![image](https://github.com/vtexdocs/dev-portal-content/assets/77292838/734ac1fb-bcfe-4c4e-8670-ac11cf3c2dc6)

Talvez mudar o <CH.Spotlight> para depois do callout faça mais sentido.